### DISCUSSION TITLE
Will there be a release of OpenSSL 1.1.1w?

### DISCUSSION THREAD

**Discussion by JerryDevis** [openssl/openssl#22023] [Q&A] [answered]
Will there be a 1.1.1w release before OpenSSL 1.1.1 EOL?

**Comment by paulidale** [comment, marked as answer]
Yes, on Monday.

**Reply by JerryDevis** [reply]
Thanks

**Comment by mattcaswell** [comment]
The announcement about this for reference:

https://mta.openssl.org/pipermail/openssl-users/2023-September/016690.html
